source/retryable-reads/tests/mapReduce-serverErrors.yml

```diff
@@ -0,0 +1,147 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
```

Comment:
This should be:
```
    - {_id: 1, x: 0}
    - {_id: 2, x: 1}
    - {_id: 3, x: 2}
```
To match the mapReduce.yml tests. Also this fixes the expected results in this file. Right now my tests are failing with: ```AssertionError: Lists differ: [{'_id': 0.0, 'value': 69.0}] != [{'_id': 0, 'value': 6}]```

Reply:
Apologies, will fix!